mod/InGameTracker/TrackerInventoryMode.cs

```diff
@@ -0,0 +1,134 @@
+        public override void EnterMode(string entryID = "", List<ShipLogFact> revealQueue = null)
+        {
+            Tracker.CheckInventory();
+            if (Wrapper == null)
+            {
+                Randomizer.OWMLModConsole.WriteLine("Wrapper is null!", OWML.Common.MessageType.Error);
+            }
+            else
+            {
+                Randomizer.OWMLModConsole.WriteLine("Opened Inventory Mode", OWML.Common.MessageType.Info);
+            }
+            Wrapper.Open();
+            Wrapper.SetName("AP Inventory");
+            Wrapper.SetItems(Tracker.InventoryItems);
+            Wrapper.SetSelectedIndex(0);
+            Wrapper.UpdateList();
+            selectedIndex = 0;
+            RootObject.name = "ArchipelagoTrackerMode";
+
+            SelectItem(0);
+        }
+
+        // Runs when the mode is closed
+        public override void ExitMode()
+        {
+            string[] keys = Tracker.NewItems.Keys.ToArray();
+            foreach (string key in keys) Tracker.NewItems[key] = false;
+            Wrapper.Close();
+        }
+
+
+        // Runs when player enters computer, update info that changes between computer sessions. Runs after EnterMode
+        public override void OnEnterComputer()
+        {
+
+        }
+
+        // Runs when the player exits the computer, after ExitMode
+        public override void OnExitComputer()
+        {
+
+        }
+
+        // Runs every frame the mode is active
+        public override void UpdateMode()
+        {
+            int changeIndex = Wrapper.UpdateList();
+
+            if (changeIndex != 0)
+            {
+                selectedIndex += changeIndex;
+
+                if (selectedIndex < 0) selectedIndex = Tracker.InventoryItems.Count - 1;
+                if (selectedIndex >= Tracker.InventoryItems.Count) selectedIndex = 0;
+
+                SelectItem(selectedIndex);
+            }
+        }
+
+        // Allows leaving the computer in this mode
+        public override bool AllowCancelInput()
+        {
+            return true;
+        }
+        
+        // Allows swapping modes while in this mode
+        public override bool AllowModeSwap()
+        {
+            return true;
+        }
+
+        // Returns the ID of the selected ship entry, used for knowing which entry should be highlighted when switching to Map Mode. Useless for us probably.
+        public override string GetFocusedEntryID()
+        {
+            return "";
+        }
+
+        // Shows the item selected and the associated info
+        private void SelectItem(int index)
+        {
+            string itemID = Tracker.ItemEntries.ElementAt(index).Key;
+            Sprite tex = TrackerManager.GetSprite(itemID);
+            bool itemExists = Enum.TryParse(itemID, out Item result);
+            // Only item that doesn't exist is the FrequencyOWV which we want to show as obtained regardless
+            if (!itemExists || Inventory[result] > 0)
```

Comment:
Looks like we're doing the same check and branch here that the descriptions file does. Maybe "descriptions" wasn't the optimal scope for that other file? I'm not too worried about this one causing bugs, so up to you if we try to fix this duplication in this PR, or leave it for when one of us feels like it.

Reply:
Yeah this could probably be refactored when I change the ItemEntries to use structs.